pageserver/src/tenant/timeline.rs

```diff
@@ -1093,6 +1109,14 @@ impl Timeline {
                     .resident_physical_size_gauge
                     .sub(layer_file_size);
 
+                self.metrics.evictions.inc();
+
+                if let Some(delta) = local_layer_residence_duration {
+                    if delta < self.conf.evictions_low_residence_duration_metric_threshold {
+                        self.metrics.evictions_with_low_residence_duration.inc();
+                    }
+                }
```

Comment:
If we add a proper abstraction for 'counter-but-only-if-below-threshold' , this logic should move into that abstraction.